helix-term/src/commands.rs

```diff
@@ -3417,8 +3411,17 @@ fn paste_impl(
         };
         (pos, pos, values.next())
     });
+    doc.apply(&transaction, view.id);
+}
 
-    Some(transaction)
+pub(crate) fn paste_bracketed_value(cx: &mut Context, contents: String) {
```

Comment:
This doesn't function as an actual command as I didn't see a good way to pass the contents through the Context. I think it's nicer to have it as a standalone function, but maybe I'm missing a way to make it work more like a regular command?

Because it's not a command, it doesn't set editor.last_insert and won't be repeated with `.`. Since you can spam your paste key as easily as `.`, I didn't think that was a big deal.